pysd/py_backend/vensim/vensim2py.py

```diff
@@ -639,6 +640,13 @@ def visit_func(self, n, vc):
             self.kind = 'component'
             return functions[n.text.lower()]
 
+        def visit_active_initial(self, n, vc):
+            string = "functions.active_initial(lambda: %(expr)s, %(init)s)" % {
+                'expr': vc[4],
+                'init': str(np.float64(vc[8]))
```

Comment:
What if the second argument of active_initial may be is expression? So this conversion will fail. Or vensim specification allow only constants? I'm not spec of vensim :)